src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -277,6 +296,24 @@
                     i18n>Retention Days must be a positive integer.</span>
             </div>
           </div>
+          <!-- Alerts -->
+          <div class="form-group row">
+            <div class="cd-col-form-label"></div>
+            <div class="cd-col-form-input">
+              <cd-alert-panel
+                type="info"
+                *ngIf="bucketForm.getValue('lock_enabled')"
+                class="me-1"
+                i18n-title>
+                  Bucket Versioning can't be disabled when Object Locking is enabled.
+              </cd-alert-panel>
+              <cd-alert-panel
+                type="warning"
+                *ngIf="bucketForm.getValue('lock_enabled')">
+                  Enabling Object Locking will allow the configuration of GOVERNANCE or COMPLIANCE modes, which will help ensure that an object version cannot be overwritten or deleted for the specified period.
+              </cd-alert-panel>
```

Comment:
Does this actually add anything not said before?

I think this is already explained in the different added `cd-hep-text`, and it's taking a lot of attention from the user as it is a warning message.

Reply:
@Pegonzal  thats totally something I copied from improvement doc.
cc @cloudbehl

Reply:
The attention for customer is important as they will not be override and delete the object.

Reply:
I'm sorry @cloudbehl, I don't fully understand your comment.

My question here is, why do we have a warning for a non destructive operation that is just repeating the same information as before, on the help-texts.

Reply:
To me what is have above is general messaging. Which they might read or might not read it. But once the checkbox is enabled we want them to be sure about what they are doing and what impact will have on their objects.

Reply:
its the repetition but I think it might be good to keep it as a warning that user would know that after this he wont be able to delete the objects for certain time period..and that he completely understands that.

Reply:
I personally still don't find it fitting